src/shared/cloudformation/utils.ts

```diff
@@ -0,0 +1,98 @@
+/*!
```

Comment:
why don't these functions live in cloudformation.ts ? This would mean a developer needs to look in an extra place when looking for cloudformation-related functions.

Reply:
I equate cloudformation.ts as a library specifically for parsing CFN files. I think a more compelling argument is that these should just live in the activation and registry files (there may have been some potential circular dependencies with the URLs, I forget if I had a more specific reason to not add to those files).

Reply:
> I equate cloudformation.ts as a library specifically for parsing CFN files.

But that is an accidental/historical association that doesn't need to continue.